src/DictVectors/initiators.jl

```diff
@@ -0,0 +1,219 @@
+"""
+    InitiatorValue{V}(; safe::V, unsafe::V, initiator::V) where V
+Composite "walker" with three fields. For use with [`InitiatorDVec`](@ref)s.
+"""
+struct InitiatorValue{V}
+    safe::V
+    unsafe::V
+    initiator::V
+end
+function InitiatorValue{V}(;safe=zero(V), unsafe=zero(V), initiator=zero(V)) where {V}
+    return InitiatorValue{V}(V(safe), V(unsafe), V(initiator))
+end
+function InitiatorValue{V}(i::InitiatorValue) where {V}
+    return InitiatorValue{V}(V(i.safe), V(i.unsafe), V(i.initiator))
+end
+
+function Base.:+(v::InitiatorValue, w::InitiatorValue)
+    return InitiatorValue(v.safe + w.safe, v.unsafe + w.unsafe, v.initiator + w.initiator)
+end
+function Base.:-(v::InitiatorValue, w::InitiatorValue)
+    return InitiatorValue(v.safe - w.safe, v.unsafe - w.unsafe, v.initiator - w.initiator)
+end
+function Base.:-(v::InitiatorValue{V}) where {V}
+    return InitiatorValue{V}(-v.safe, -v.unsafe, -v.initiator)
+end
+Base.zero(::Union{I,Type{I}}) where {V,I<:InitiatorValue{V}} = InitiatorValue{V}()
+
+Base.convert(::Type{<:InitiatorValue{V}}, x::InitiatorValue{V}) where {V} = x
+function Base.convert(::Type{<:InitiatorValue{U}}, x::InitiatorValue{V}) where {U,V}
+    return InitiatorValue{U}(safe=x.safe, unsafe=x.unsafe, initiator=x.initiator)
+end
+function Base.convert(::Type{<:InitiatorValue{V}}, x) where {V}
+    return InitiatorValue{V}(safe=convert(V, x))
+end
```

Comment:
This code looks like it is duplicated, as it appears in `initiatorvalue.jl` as well.

Reply:
Actually, not the `convert` methods.

Reply:
Oh, I see. `initiatorvalue.jl` was meant to be deleted. Fixed.